src/ep_worker.c

```diff
@@ -9848,14 +9855,17 @@ static ep_stat_t w_form_query_getall_insert_new(worker_data_t *wd, obj_info_t *o
         strcat_safe(query, "], ", query_size);
     }
     /* The last index is calculated in the object's table. */
+
+    strcat_safe(query, "IFNULL('". query_size);
+    strcat_safe(query, newrow->be_keys[backend_index], query_size);
```

Comment:
1) You should take into account that `backend_index` may be equal `-1` here
2) It's unclear from code what `newrow->be_keys[backend_index]` means
    I suggest to add explanation comment for that